src/api/app/controllers/trigger_workflow_controller.rb

```diff
@@ -83,6 +83,6 @@ def extract_scm_webhook
 
     # There are plenty of different pull/merge request action which we don't support.
```

Comment:
This comment should be updated to reflect the changes.

Reply:
Updated the comment